eventbridge_api_destination_rule/main.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_cloudwatch_event_rule" "rule" {
+  name          = var.name
+  description   = var.description
```

Comment:
Worth adding an `event_bus_name` variable to allow rules to be used on custom event buses. Default is `default`. Will also need adding to the target.

Reply:
Yeah good point. I've added a variable in with a default of default.